src/game/sa1_sa2_shared/collision.c

```diff
@@ -90,7 +90,7 @@ u32 Player_CrumblingPlatformCollision(Sprite *s, s32 sx, s32 sy, Player *p)
     return result;
 }
 
-bool32 Player_HitboxCollision(Sprite *s, s32 sx, s32 sy, s16 hbIndex, Player *p, s16 hbIndexPlayer)
+bool32 Coll_Sprite_Player_HitboxIntersecting(Sprite *s, s32 sx, s32 sy, s16 hbIndex, Player *p, s16 hbIndexPlayer)
```

Comment:
Maybe `Coll_Player_Entity_HitboxN` or something like that?
Something that underlines that a specific hitbox gets checked?

Reply:
Good idea!